ansible/README.md

```diff
@@ -0,0 +1,189 @@
+
+Use the official [Ansible installation guide](http://docs.ansible.com/ansible/intro_installation.html) to install Ansible. Here are a few examples on basic installation commands:
+
+Ubuntu/Debian:
+```
+sudo apt-get install ansible
+```
+
+CentOS/RedHat:
+```
+sudo yum install epel-release
+sudo yum install ansible
+```
+
+Mac OSX:
+```
+sudo easy_install pip
+sudo pip install ansible
+```
+
+To make life easier, you can start an SSH Agent and load your SSH key(s). This way ansible will have an uninterrupted way of connecting to your servers.
+
+```
+ssh-agent > ~/.ssh/ssh.env
+source ~/.ssh/ssh.env
+
+ssh-add private.key
+```
+
+Subsequently, as long as the agent is running, you can use `source ~/.ssh/ssh.env` to load the keys to the current session.
+
+### Optional cloud dependencies
+
+If you are using a cloud provider to host your servers, you need the below dependencies installed on your local machine.
+
+#### DigitalOcean inventory dependencies:
+
+Ubuntu/Debian:
+```
+sudo apt-get install python-pip
+sudo pip install dopy
+```
+
+CentOS/RedHat:
+```
+sudo yum install python-pip
+sudo pip install dopy
+```
+
+Mac OSX:
+```
+sudo pip install dopy
+```
+
+#### Amazon AWS inventory dependencies:
+
+Ubuntu/Debian:
+```
+sudo apt-get install python-boto
+```
+
+CentOS/RedHat:
+```
+sudo yum install python-boto
+```
+
+Mac OSX:
+```
+sudo pip install boto
+```
+
+## Refreshing the DigitalOcean inventory
+
+If you just finished creating droplets, the local DigitalOcean inventory cache is not up-to-date. To refresh it, run:
+
+```
+DO_API_TOKEN="<The API token received from DigitalOcean>"
+python -u inventory/digital_ocean.py --refresh-cache 1> /dev/null
+```
+
+## Refreshing the Amazon AWS inventory
+
+If you just finished creating Amazon AWS EC2 instances, the local AWS inventory cache is not up-to-date. To refresh it, run:
+
+```
+AWS_ACCESS_KEY_ID='<The API access key ID received from Amazon>'
+AWS_SECRET_ACCESS_KEY='<The API secret access key received from Amazon>'
+python -u inventory/ec2.py --refresh-cache 1> /dev/null
+```
+
+Note: you don't need the access key and secret key set, if you are running ansible on an Amazon AMI instance with the proper IAM permissions set.
+
+## Running the playbook
+
+The playbook is locked down to only run if the environment variable `TF_VAR_TESTNET_NAME` is populated. This is a precaution so you don't accidentally run the playbook on all your servers.
+
+The variable `TF_VAR_TESTNET_NAME` contains the testnet name which ansible translates into an ansible group. If you used Terraform to create the servers, it was the testnet name used there.
+
+If the playbook cannot connect to the servers because of public key denial, your SSH Agent is not set up properly. Alternatively you can add the SSH key to ansible using the `--private-key` option.
+
+### DigitalOcean
```

Comment:
Why would you run ansible on your local machine? The installed package already set up a single-node validator and ansible would set up the same thing.